lib/rpmsg/rpmsg_virtio.c

```diff
@@ -867,6 +885,7 @@ void rpmsg_deinit_vdev(struct rpmsg_virtio_device *rvdev)
 		rvdev->rvq = 0;
 		rvdev->svq = 0;
 
+		rpmsg_virtio_delete_virtqueues(rvdev);
```

Comment:
`virtio_delete_virtqueues(struct virtio_device *vdev)` returns -ENXIO if `vdev->func->delete_virtqueues` not defined.
But delete_virtqueues is optional.

To be compatible, seems that the virtio_delete_virtqueues should be reworked  to suppress the return .
I pushed a PR in this way to fix this before the release

https://github.com/OpenAMP/open-amp/pull/527

could you review it Please

Reply:
Thanks, LGTM